rollup.config.js

```diff
@@ -99,12 +99,20 @@ export default [
 			json(),
 			ts_plugin
 		],
-		output: {
-			file: 'compiler.js',
-			format: is_publish ? 'umd' : 'cjs',
-			name: 'svelte',
-			sourcemap: true,
-		},
+		output: [
+			{
+				file: 'compiler.js',
+				format: is_publish ? 'umd' : 'cjs',
+				name: 'svelte',
+				sourcemap: true,
+			},
+			{
+				file: 'compiler.mjs',
+				format: 'esm',
+				name: 'svelte_esm',
```

Comment:
why is the name different? and why disable sourcemap?

Reply:
Well, I'm no Rollup expert, I thought names should've been different.

I'll change the sourcemap, indeed it's better to keep it :)

Reply:
Here it is, changed 😉

Reply:
Thanks for changing the name to be the same. That looks to be the right thing to do to me based on the Rollup docs for that option: https://rollupjs.org/guide/en/#outputname